view_component-fragment_caching.gemspec

```
@@ -22,7 +22,7 @@ require_relative 'lib/view_component/fragment_caching/version'
     end
 
   spec.add_dependency 'rails', '~> 7.0'
-  spec.add_dependency 'view_component', '~> 2.43'
+  spec.add_dependency 'view_component', '>= 3.9', '< 4.0'
```

How would you feel about using the twiddle wakka here? I would be concerned about future breaking changes with VC 3.10+ around caching that may cause some unexpected behavior.
```suggestion
  spec.add_dependency 'view_component', '~> 3.9'
```

---

Good point! Pushed with the twiddle wakka 👍